samples/Logging/AnyMethodHandleItAttribute.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using AspectInjector.Broker;
+using Aspects.Universal.Aspects;
+using Aspects.Universal.Attributes;
+using Aspects.Universal.Events;
+
+namespace Logging
+{
+    [Injection(typeof(MethodWrapperAspect))]

```

Comment:
It should work without this attribute, because it is set on the base class

Reply:
You need Inherited=true param. Take a look here https://github.com/pamidur/aspect-injector/blob/c29270fa922deefc4d24f2271385dbff5ea3a4cb/samples/src/Cache/CacheAspect.cs#L11

Reply:
I see. This is fixed.